src/modules/Task.js

```diff
@@ -0,0 +1,99 @@
+import Activity from './activity.js';
+/* eslint-disable-next-line no-unused-vars */
```

Comment:
- [ ] Kindly ensure you dont disable linter rules in your codebase because linters help to ensure that you write a bug-free, clean, and workable code. Always ensure you fix your linter errors instead of disabling them.